asset/dcr/loader.go

```diff
@@ -62,7 +62,20 @@ func CreateWallet(ctx context.Context, params asset.CreateWalletParams, recovery
 		birthday = time.Now()
 	}
 
-	wb, err := asset.NewWalletBase(params.OpenWalletParams, seed, params.Pass, birthday, walletTraits)
+	// Adjust seed to create the same wallet as dex.
+	b := make([]byte, len(seed)+4)
+	copy(b, seed)
+	binary.BigEndian.PutUint32(b[len(seed):], 42)
+	tweakedSeed := blake256.Sum256(b)
+
+	_, _, _, acctKeySLIP0044Priv, err := udb.HDKeysFromSeed(tweakedSeed[:], chainParams)
+	if err != nil {
+		return nil, err
+	}
+	defer acctKeySLIP0044Priv.Zero()
+	xpub := acctKeySLIP0044Priv.Neuter()
```

Comment:
would this make more sense to do in NewWalletBase?

Reply:
Yes, and I started there, but the chaincfg.Params is decred specific so it got a little weird. I want to rework this so that it's only for decred at some point then there doesn't need to be that abstracting layer.